media/static-bundles.json

```diff
@@ -990,6 +996,20 @@
       ],
       "name": "firefox_whatsnew_87_eu_vpn"
     },
+    {
+      "files": [
+        "js/firefox/whatsnew/up-to-date.js",
+        "js/firefox/whatsnew/whatsnew-88-en.js"
+      ],
+      "name": "firefox_whatsnew_88_en"
+    },
+    {
+      "files": [
+        "js/libs/mozilla-traffic-cop.js",
+        "js/firefox/whatsnew/whatsnew-88-exp.js"
+      ],
+      "name": "firefox_whatsnew_88_exp"
+    },
```

Comment:
These bundles do not overlap, so every WNP 88 will need `firefox_whatsnew_88_en` regardless if they're enrolled in the experiment or not.